packages/components/src/internal/components/domainproperties/samples/UniqueIdBanner.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, memo, useCallback } from 'react';
+import { SampleTypeModel } from './models';
+import { Alert } from '../../base/Alert';
+import { Button } from 'react-bootstrap';
+import { IDomainField } from '../models';
+import { UNIQUE_ID_TYPE } from '../PropDescType';
+
+interface Props  {
+    model: SampleTypeModel
+    isFieldsPanel: boolean
+    onAddField: (fieldConfig: Partial<IDomainField>) => void
+}
+
+export const DEFAULT_UNIQUE_ID_FIELD = {
+    dataType: UNIQUE_ID_TYPE,
+    conceptURI: UNIQUE_ID_TYPE.conceptURI,
+    rangeURI: UNIQUE_ID_TYPE.rangeURI,
+    name: 'Barcode',
```

Comment:
could that `shownInInsertView` value be added here or is there a case where we wouldn't set that for the new unique id field?